scss-lint.yml

```diff
@@ -215,7 +215,7 @@ linters:
 
   StringQuotes:
     enabled: true
-    style: single_quotes # or double_quotes
+    style: double_quotes # or single_quotes
```

Comment:
all of our stylesheets use double quotes. It would be easier to change the rule than to change every stylesheet in every codebase